python/sglang/srt/disaggregation/prefill.py

```diff
@@ -115,7 +117,16 @@ def _init_kv_manager(self) -> BaseKVManager:
         return kv_manager
 
     def add(self, req: Req) -> None:
-        kv_sender_class = get_kv_class(self.transfer_backend, KVClassType.SENDER)
+        if (
+            req.bootstrap_room == FakeBootstrapRoom
```

Comment:
we can just use `bootstrap_room` to determine warmup request because LB always gives non negative room id

Reply:
as we  talked in slac. this bootstrap_room can't be fake , we should use fake bootstrap host only